beacon-chain/core/helpers/committee.go

```diff
@@ -648,3 +677,17 @@ func toCommitteeCache(slot uint64, crosslinkCommittees []*CrosslinkCommittee) *c
 
 	return committees
 }
+
+// toCrosslinkCommittee coverts a cache'd committee object
+// into a regular crosslink committee object.
+func toCrosslinkCommittee(cachedCommittee *cache.CommitteesInSlot) []*CrosslinkCommittee {
```

Comment:
cant we save it like this in the cache ? it seems wasteful to convert it everytime we use the cache.